static/app/components/feedback/useFetchFeedbackInfiniteListData.tsx

```diff
@@ -41,17 +41,19 @@ export const EMPTY_INFINITE_LIST_DATA: ReturnType<
   issues: [],
   loadMoreRows: () => Promise.resolve(),
   setFeedback: () => undefined,
+  setQueryStatus: () => undefined,
 };
 
 export default function useFetchFeedbackInfiniteListData({queryView}: Params) {
   const organization = useOrganization();
+  const [queryStatus, setQueryStatus] = useState('unresolved');
 
   const query = useMemo(
     () => ({
       ...queryView,
-      query: 'issue.category:feedback ' + queryView.query,
+      query: 'issue.category:feedback ' + 'status:' + queryStatus + queryView.query,
```

Comment:
small nit here, string templates might make this a bit more readable here, esp with multiple strings/vars (e.g. spaces are easier to get lost) 

```suggestion
      query: `issue.category:feedback status:${queryStatus} ${queryView.query}`,
```

Reply:
good catch! ryan implemented this change in https://github.com/getsentry/sentry/pull/58804 thankfully